networkmanager/networkmanager.go

```diff
@@ -258,7 +266,12 @@ func (manager *NetworkManager) AddInstanceToNetwork(instanceID, networkID string
 		}
 	}()
 
-	netConfig, runtimeConfig, err := manager.prepareNetRuntimeConfig(instanceID, networkID, ipSubnet, params)
+	hosts, err := manager.prepareHostnameList(networkID, params)
```

Comment:
You can move prepareHostnameList inside prepareCNIConfig. Because params, hosts duplicates in some sense.

Reply:
Done